ios/.npmignore

```diff
@@ -0,0 +1,3 @@
+# OS X
+# Sometimes these are hiding in the iOS frameworks
+.DS_Store
```

Comment:
As I can see `.DS_Store` is already specified in the `.gitignore` file. And I think that's enough. I don't see any need to add the `.npmignore` file for that. BTW, sometimes `.npmignore` and `.gitignore` files don't play nice together.

But to be honest I'm not sure. You need to double check it. I guess If you remove the `.npmignore` file and run the `npm pack --dry-run` command, you won't see `.DS_Store` in the output.

Everything else looks good to me! 🙂

Reply:
In my tests the `.DS_Store` file was not being removed unless it was specified in the `.npmignore` file as well, I think because adding the framework in the "files" section means all the files within the framework are automatically added unless explicitly marked to be ignored in a subdirectory `.npmignore` file.

Reply:
Makes sense 🙂

Reply:
Sorry, Ian, I didn't notice that at least 1 approving review for this PR is required. LGTM. But I think it will be great if @j-stokes takes a quick look at this PR too.